erc20-load-test-tool/script/run-dapp.py

```diff
@@ -0,0 +1,206 @@
+        {
+            "from": contract_deploy_account,
+            "gas": gas,
+            "gasPrice": gas_price,
+            "nonce": w3.eth.get_transaction_count(contract_deploy_account),
+            "chainId": chainid,
+        }
+    )
+    signed_txn = w3.eth.account.sign_transaction(
+        construct_txn, contract_deploy_account_key)
+    # Validating transaction hash
+    tx_hash_send = signed_txn.hash
+    tx_hash = w3.eth.send_raw_transaction(signed_txn.rawTransaction)
+    assert tx_hash_send == tx_hash, "smart contract deploy transaction hash mismatch error"
+    print("smart contract deploy transaction hash successfully validated")
+    # Validating transaction receipt
+    tx_receipt = w3.eth.wait_for_transaction_receipt(tx_hash)
+    print("smart contract deploy transaction receipt: '{}'".format(tx_receipt))
+    if tx_receipt.status == 1:
+        print(
+            "smart contract deploy success, contract address: '{}'".format(
+                tx_receipt.contractAddress
+            )
+        )
+        contract_deploy_status = True
+    else:
+        assert contract_deploy_status, "smart contract deploy failed"
+
+    contractAddress = tx_receipt.contractAddress
+
+    erc20_contract = w3.eth.contract(address=contractAddress, abi=abi1)
+
+    erc20_item_count = erc20_contract.functions.balanceOf(
+        contract_deploy_account).call()
+    print("erc20_item_count account: {} {} \n".format(
+        contract_deploy_account, erc20_item_count))
+
+    # distributing tokens to all senders
+    for i in range(1, len(accts)):
+        construct_txn = erc20_contract.functions.transfer(accts[i], 10000000).buildTransaction({
+            'from': contract_deploy_account,
+            'gas': gas,
+            'gasPrice': gas_price,
+            'nonce': w3.eth.get_transaction_count(contract_deploy_account),
+            'chainId': chainid
+        })
+        signed_txn = w3.eth.account.sign_transaction(
+            construct_txn, contract_deploy_account_key)
+        # Validating transaction hash
+        tx_hash_send = signed_txn.hash
+        tx_hash = w3.eth.send_raw_transaction(signed_txn.rawTransaction)
+        assert tx_hash_send == tx_hash, "tx hash mismatch"
+        # Validating transaction receipt
+        tx_receipt = w3.eth.wait_for_transaction_receipt(tx_hash)
+        print("transaction receipt: '{}'".format(tx_receipt))
+        assert tx_receipt.status == 1, "transaction failed"
+
+        erc20_item_count = erc20_contract.functions.balanceOf(accts[i]).call()
+        print("erc20_item_count account: {} {} \n".format(
+            accts[i], erc20_item_count))
+
+    return contractAddress
+
+
+def run_erc20(priv_key, contract_address, i):
+    # run erc20 app
+    global port
+    print("start "+str(port+i))
+    if(share_contract):
+        mvn = "cd .. ; mvn spring-boot:run -Dspring-boot.run.arguments='--server.port=" + \
```

Comment:
I guess you can avoid command repetition. How about exporting TOKEN_CONTRACT_ADDRESS rather than if/else?